composer.json

```diff
@@ -46,6 +47,7 @@
         "preferred-install": "dist"
     },
     "extra": {
+        "contao-manager-plugin": "Contao\\FaqBundle\\ContaoManager\\Plugin",
```

Comment:
I am still talking about making the plugin a convention and the manager simply assuming that it is there instead of checking the composer configuration.